source/upgrade/v3.txt

```diff
@@ -97,7 +97,16 @@ Version 3.0 Breaking Changes
   To configure logging in your application, see the :ref:`csharp-logging` guide.
 
 - The LINQ2 provider has been removed from this version of the driver.
-  You must use LINQ3 for all LINQ queries.
+  You must use LINQ3 for all LINQ queries. This change has the following effect on the
```

Comment:
S: since there's only one item in this nested list, it might make more sense to just combine it all into one paragraph? Or add a line break after "You must use LINQ3 for all LINQ queries" and start a new paragraph with "Queries that use client-side projections..."